test/object-store/object.cpp

```diff
@@ -48,6 +48,8 @@ std::vector<T> get_vector(std::initializer_list<T> list)
 {
     return std::vector<T>(list);
 }
+// Used for unit testing CreatePolicy's
+static CreatePolicy last_create_policy;
```

Comment:
Given that this is just a one-off test it's probably not worth doing anything complicated to avoid the global, but this should at least be a static member of the class which uses it.